diff -urN yawp-0.4.3/applet/plasma-applet-yawp.desktop.in yawp-0.4.3-patched/applet/plasma-applet-yawp.desktop.in
--- yawp-0.4.3/applet/plasma-applet-yawp.desktop.in	2012-04-17 22:17:36.000000000 +0400
+++ yawp-0.4.3-patched/applet/plasma-applet-yawp.desktop.in	2012-07-19 06:28:35.024477807 +0400
@@ -18,4 +18,5 @@
 X-Plasma-NotificationArea=true
 
 Comment[cs]=Ještě jeden plasmoid zobrazující počasí
-Comment[sk]=Ďaľší plasmoid zobrazujúci počasie
+Comment[ru]=Еще один плазмоид погоды
+Comment[sk]=Ďaľší plasmoid zobrazujúci počasie
\ В конце файла нет новой строки
diff -urN yawp-0.4.3/ions/ion-accuweather.desktop yawp-0.4.3-patched/ions/ion-accuweather.desktop
--- yawp-0.4.3/ions/ion-accuweather.desktop	2012-04-17 22:17:36.000000000 +0400
+++ yawp-0.4.3-patched/ions/ion-accuweather.desktop	2012-07-19 06:27:57.731944028 +0400
@@ -3,6 +3,8 @@
 Comment=XML Data from AccuWeather
 Name[cs]=Počasí Accuweather.com
 Comment[cs]=XML data od Accuweather.com
+Name[ru]=Погодная служба AccuWeather
+Comment[ru]=XML данные от AccuWeather
 Name[sk]=Počasie Accuweather.com
 Comment[sk]=XML dáta od Accuweather.com
 X-KDE-ServiceTypes=WeatherEngine/Ion
diff -urN yawp-0.4.3/ions/ion-google.desktop yawp-0.4.3-patched/ions/ion-google.desktop
--- yawp-0.4.3/ions/ion-google.desktop	2012-04-17 22:17:36.000000000 +0400
+++ yawp-0.4.3-patched/ions/ion-google.desktop	2012-07-19 06:29:38.540683746 +0400
@@ -3,6 +3,8 @@
 Comment=XML Data from Google
 Name[cs]=Počasí Google
 Comment[cs]=XML data od Google
+Name[ru]=Служба погоды от Google
+Comment[ru]=XML данные от Google
 Name[sk]=Počasie Google
 Comment[sk]=XML dáta od Google
 X-KDE-ServiceTypes=WeatherEngine/Ion
diff -urN yawp-0.4.3/ions/ion-wunderground.desktop yawp-0.4.3-patched/ions/ion-wunderground.desktop
--- yawp-0.4.3/ions/ion-wunderground.desktop	2012-04-17 22:17:36.000000000 +0400
+++ yawp-0.4.3-patched/ions/ion-wunderground.desktop	2012-07-19 06:30:37.699944152 +0400
@@ -1,6 +1,8 @@
 [Desktop Entry]
 Name=Wunderground Weather Service
 Comment=XML Data from Wunderground
+Name[ru]=Служба погоды Wunderground
+Comment[ru]=XML данные от Wunderground
 Name[sk]=Počasie Wunderground
 Comment[sk]=XML dáta od Wunderground
 X-KDE-ServiceTypes=WeatherEngine/Ion
diff -urN yawp-0.4.3/ions/kde43/ion-accuweather.desktop yawp-0.4.3-patched/ions/kde43/ion-accuweather.desktop
--- yawp-0.4.3/ions/kde43/ion-accuweather.desktop	2012-04-17 22:17:36.000000000 +0400
+++ yawp-0.4.3-patched/ions/kde43/ion-accuweather.desktop	2012-07-19 06:31:33.719243816 +0400
@@ -3,6 +3,8 @@
 Comment=XML Data from AccuWeather
 Name[cs]=Počasí Accuweather.com
 Comment[cs]=XML data od Accuweather.com
+Name[ru]=Погодная служба AccuWeather
+Comment[ru]=XML данные от AccuWeather
 Name[sk]=Počasie Accuweather.com
 Comment[sk]=XML dáta od Accuweather.com
 X-KDE-ServiceTypes=Plasma/DataEngine
diff -urN yawp-0.4.3/ions/kde43/ion-google.desktop yawp-0.4.3-patched/ions/kde43/ion-google.desktop
--- yawp-0.4.3/ions/kde43/ion-google.desktop	2012-04-17 22:17:36.000000000 +0400
+++ yawp-0.4.3-patched/ions/kde43/ion-google.desktop	2012-07-19 06:32:41.473396773 +0400
@@ -3,6 +3,8 @@
 Comment=XML Data from Google
 Name[cs]=Počasí Google
 Comment[cs]=XML data od Google
+Name[ru]=Служба погоды от Google
+Comment[ru]=XML данные от Google
 Name[sk]=Počasie Google
 Comment[sk]=XML dáta od Google
 X-KDE-ServiceTypes=Plasma/DataEngine
diff -urN yawp-0.4.3/ions/kde43/ion-wunderground.desktop yawp-0.4.3-patched/ions/kde43/ion-wunderground.desktop
--- yawp-0.4.3/ions/kde43/ion-wunderground.desktop	2012-04-17 22:17:36.000000000 +0400
+++ yawp-0.4.3-patched/ions/kde43/ion-wunderground.desktop	2012-07-19 06:33:29.041802085 +0400
@@ -1,6 +1,8 @@
 [Desktop Entry]
 Name=Wunderground Weather Service
 Comment=XML Data from Wunderground
+Name[ru]=Служба погоды Wunderground
+Comment[ru]=XML данные от Wunderground
 Name[sk]=Počasie Wunderground
 Comment[sk]=XML dáta od Wunderground
 X-KDE-ServiceTypes=Plasma/DataEngine
